utility/src/main/java/org/oppia/util/statusbar/StatusBarColor.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.util.statusbar
+
+import android.os.Build
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+
+class StatusBarColor {
+  companion object {
+    fun statusBarColorUpdate(colorId: Int, activity: AppCompatActivity, statusBarLight: Boolean) {
```

Comment:
Add a kdoc for this functions mentioning all its params and it does.

Reply:
Done

Reply:
+1